oic.r.cred.raml

```diff
@@ -82,8 +81,7 @@ traits:
                     "subjectuuid": "e61c3e6b-9c54-4b81-8ce5-f9039c1d04d9",
```

Comment:
This was an error from before but the first cred had a "credid" of 55 as well.  Is it allowed to have the same "credid" for two creds?  I think it must be a unique number, right?  I think line 81 should use a different "credid".